src/org/usfirst/frc/team1076/robot/commands/RotateCommand.java

```diff
@@ -12,28 +12,38 @@
 
 	LeftRightMotors leftRight;
 	FrontBackMotors frontBack;
-	double angle;
+	double targetTime;
+	double time;
+	double speed;
 	
 	/**
 	 * Creates a RotateCommand giving it the subsystems and angle it requires.
 	 * @param leftRight
 	 * @param frontBack
-	 * @param angle measured in degrees clockwise.
+	 * @param targetTime measured in seconds.
+	 * @param speed in the range of -1 to 1. Positive numbers rotate clockwise, negative numbers rotate counterclockwise.
 	 */
-    public RotateCommand(LeftRightMotors leftRight, FrontBackMotors frontBack, double angle) {
+    public RotateCommand(LeftRightMotors leftRight, FrontBackMotors frontBack, double targetTime, double speed) {
     	requires(leftRight);
     	requires(frontBack);
     	this.leftRight = leftRight;
     	this.frontBack = frontBack;
-    	this.angle = angle;
+    	this.targetTime = targetTime;
+    	this.speed = speed;
     }
 
     // Called just before this Command runs the first time
     protected void initialize() {
+    	this.time = 0;
     }
 
     // Called repeatedly when this Command is scheduled to run
     protected void execute() {
+    	time +=1/50.0;
```

Comment:
Yeah, this is what we're doing in all the other commands, we can do it in a more elegant way on the real robot if we want, but sticking with this would be good for consistency.